src/open_inwoner/templates/pages/cases/status_inner.html

```diff
@@ -73,6 +73,49 @@ <h2 class="h2 {% if case.new_docs %}indicator{% endif %}" id="documents">{% tran
                 </section>
             {% endif %}
 
+            {# Questions/contactmomenten #}
```

Comment:
I think it would be a bit nicer to add an include here? So Something like `{% include 'pages/cases/questions.html' %}` This way it will be easier to swap the order of the contact-moment form with this new part, because in the design you can see they need to be reversed 
(questions first, then the contact-form all the way at the bottom of the page):

Also the contact-form should now be 100% the width of the container/column.

<img width="903" alt="Screenshot 2024-03-04 at 16 43 38" src="https://github.com/maykinmedia/open-inwoner/assets/118177951/d6b7da44-2286-4d9c-b0ed-8e68f8487840">

Reply:
There is no contact form on this page, perhaps you're thinking of the vragen list view? Anyways, the suggestion sounds reasonable, if only to prevent the case detail template from growing too big.